test/parallel/test-http2-shutdown-errors.js

```diff
@@ -29,7 +29,7 @@ const tests = Object.getOwnPropertyNames(constants)
 let currentError;
 
 // mock submitGoaway because we only care about testing error handling
-Http2Session.prototype.submitGoaway = () => currentError.ngError;
+Http2Session.prototype.goaway = () => currentError.ngError;
```

Comment:
nit: `goAway` to be consistent with `pushPromise`, `rstStream`.

Reply:
The spec treats it as `GOAWAY` and `PUSH_PROMISE` / `RST_STREAM`, I was segmenting based on the presence of the `_`